public/locales/en/common.json

```diff
@@ -90,9 +90,9 @@
       "to-vault": "To vault",
       "to-wallet": "To wallet",
       "token-input": {
-        "search-select-token": "Select a Token",
+        "search-select-token": "Select a token to deposit",
```

Comment:
Since the component is reused in many places, we either need to use a generic text or place custom ones with conditionals.

Here what happens on withdrawals:
![image](https://user-images.githubusercontent.com/18649057/169628123-e55241df-aa77-4c07-b755-ee5070ca5d29.png)

Reply:
we prob need to look into our components - they seem to be forcing us to reduce the guidance we can give to ppl

Reply:
ideally we need to add conditional renders for different messages - will create an epic for it